List menu -> radial menu conversions

MuckerMayhem opened the pull request:
:cl: Mucker
rscadd: The RCD now uses a radial menu for swapping modes.
rscadd: Surgery steps now use a radial menu for picking organs.
rscadd: The holstering verb now uses a radial menu to choose a holster if there are more than one.
/:cl:

SierraKomodo commented on code/datums/extensions/holster/holster.dm:
```diff
@@ -117,15 +111,49 @@
+			holsters += item.get_holsters()
+			continue
+
+		if (istype(item, /obj/item/clothing))
+			var/obj/item/clothing/C = item
+
+			if (length(C.accessories))
+				for (var/obj/item/clothing/accessory/A in C.accessories)
+					if (has_extension(A, /datum/extension/holster))
+						holsters += A.get_holsters()
+						continue
+
```
I believe this is already covered by clothing's `get_holsters()` override, which would be run by the block above.

SierraKomodo commented on code/datums/extensions/holster/holster.dm:
```diff
@@ -117,15 +111,49 @@
+
+
+	if (!length(holsters))
+		to_chat(usr, SPAN_WARNING("You have no holster(s) equipped!"))
```
```suggestion
		to_chat(usr, SPAN_WARNING("You have no holsters equipped!"))
```

SierraKomodo commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -93,8 +93,34 @@
+	)
+	var/choice = show_radial_menu(user, src, options, require_near = TRUE, radius = 42, tooltips = TRUE, check_locs = list(src))
+
+	if (!choice)
```
```suggestion
	if (!choice || !user.use_sanity_check(src))
```

MuckerMayhem committed:
holster radial menu

MuckerMayhem committed:
rcd radial menu

MuckerMayhem committed:
surgery radial menus

Spookerton reviewed (changes requested):
needs some sanity on the other checks too

![https://cdn.discordapp.com/attachments/492799900145418270/1128285626005987418/dreamseeker_O6e0glsZiq.jpg](https://cdn.discordapp.com/attachments/492799900145418270/1128285626005987418/dreamseeker_O6e0glsZiq.jpg)

Spookerton committed:
use_sanity_check radial menu corrections

also removes auto-choice from surgery radials - this allows preventing
an action you don't actually want to take
